website/docs/docs/build/metrics.md

```diff
@@ -406,11 +487,72 @@ You may find some pieces of functionality, like secondary calculations, complica
 
+The supported Secondary Calculations are:
+
+#### Period over Period:
+
+The period over period secondary calculation performs a calculation against the metric(s) in question by either determining the difference or the ratio between two points of time. This other point in time is determined by the input variable which looks at the grain selected in the macro. 
+
+| Input                  | Example | Description | Required |
+| -------------------------- | ----------- | ----------- | -----------|
+| `comparison_strategy`      | `ratio` or `difference` | How to calculate the delta between the two periods | Yes |
+| `interval`                 | 1 | Integer - the number of time grains to look back | Yes |
+| `alias`                    | `week_over_week` | The column alias for the resulting calculation | No |
+| `metric_list`              | `base_sum_metric` | List of metrics that the secondary calculation should be applied to. Default is all metrics selected | No |
+
+#### Period to Date:
+
+The period to date secondary calculation performs an aggregation on a defined **period** of time that is equal to or coarser (higher, more aggregated) than the grain selected. Great example of this is when you want to display a month_to_date value alongside your weekly grained metric.
+
+| Input                  | Example | Description | Required |
+| -------------------------- | ----------- | ----------- | -----------|
+| `aggregate`                | `max`, `average` | The aggregation to use in the window function. Options vary based on the primary aggregation and are enforced in [validate_aggregate_coherence()](https://github.com/dbt-labs/dbt_metrics/blob/main/macros/validation/validate_aggregate_coherence.sql). | Yes |
+| `period`                   | `"day"`, `"week"` | The time grain to aggregate to. One of [`"day"`, `"week"`, `"month"`, `"quarter"`, `"year"`]. Must be at equal or coarser (higher, more aggregated) granularity than the metric's grain (see [Time Grains](#time-grains) below). In example grain of `month`, the acceptable periods would be `month`, `quarter`, or `year`. | Yes |
+| `alias`                    | `month_to_date` | The column alias for the resulting calculation | No |
+| `metric_list`              | `base_sum_metric` | List of metrics that the secondary calculation should be applied to. Default is all metrics selected | No |
+
+#### Rolling:
+
+<VersionBlock firstVersion="1.3" >
+The rolling secondary calculation performs an aggregation on a number of rows in metric dataset. For example, if the user selects the `week` grain and sets a rolling secondary calculation to `4` then the value returned will be a rolling 4 week calculation of whatever aggregation type was selected. If the `interval` input is not provided then the rolling caclulation will be unbounded on all preceding rows.
```

Comment:
```suggestion
The rolling secondary calculation performs an aggregation on a number of rows in the metric dataset. For example, if the user selects the `week` grain and sets a rolling secondary calculation to `4`, then the value returned will be a rolling 4-week calculation of whatever aggregation type was selected. If the `interval` input is not provided, then the rolling calculation will return unlimited on all preceding rows. 


```

Reply:
I'm going to resolve this one because unbounded is the more correct terminology.